### DISCUSSION TITLE
VSCode Browser IDE auto-open

### DISCUSSION THREAD

**Discussion by mkyrilov** [loft-sh/devpod#1223] [Q&A] [answered]
When I open a workspace and I select the `VSCode Browser` IDE option, it automatically opens up in my web browser. I would prefer if it doesn't do that. Ideally I would be able to copy the URL and manually open that in that in a browser of my choosing (not my default browser).
Is it possible to achieve this?

**Comment by amithm7** [comment]
I think you can start the server manually. VS Code server binary is installed here:

```sh
~/.vscode-server/cli/servers/Stable-$COMMIT_ID/server/bin/code-server --help
```

**Comment by pascalbreuninger** [comment, marked as answer]
Hey @mkyrilov, you can set this behaviour per IDE by running `devpod ide set-options openvscode -o OPEN=false`
